services_app/build.gradle

```diff
@@ -30,10 +30,11 @@ android {
         applicationId(groupId + servicesName)
         minSdkVersion(minVersion)
         targetSdkVersion(targetVersion)
-        versionCode(releaseVersionCode)
+        versionCode releaseVersionCode
         versionName(versionCodeName)
         testInstrumentationRunner(instrumentationRunner)
-		android.defaultConfig.vectorDrawables.useSupportLibrary = true
+        signingConfig signingConfigs.debug
```

Comment:
why do you need this line? It seems like something gradle should automatically assign that when you specify to gradle to either build with debug or release? It seems like you are hard-coding in something we may not want when releasing.